docs/sitegen.yaml

```diff
@@ -52,26 +52,20 @@ backend:
         type: "image"
         value: "images/helidon_logo_dark.svg"
       items:
+        - type: "PAGE"
+          title: "Introduction"
+          source: "about/introduction.adoc"
+          glyph:
+            type: "icon"
+            value: "widgets"
+        - type: "PAGE"
```

Comment:
We should have a top level"Get Started" page here with some version of what is currently in (the misnamed) `prerequisites.adoc`. It's important that we have "Get Started" very visible at the top level like we have in 2.x.

Reply:
We are going in direction of separating Helidon documentation into self-contained documentation for each flavor. My idea is that all QuickStart pages must be flavor specific. Having one page in the top level which explains how to add `JAVA_HOME` and contains links to SE and MP quickstart guides doesn't make sense to me.